community.dbuild

```diff
@@ -984,4 +989,60 @@ build += {
     extra.exclude: ["plugin"]  // we never build sbt plugins
   }
 
+  // forked to remove the usual bintray gunk
+  ${vars.base} {
+    name: "scalameta"
+    uri:  ${vars.uris.scalameta-uri}
+    // else, bintray stuff goes boom
+    extra.options: ["-Dsbt.prohibit.publish=true"]
+    extra.exclude: ["readme"]  // requires scalatex-site which requires Scala.js
+  }
+
+  // forked to remove wartremover and coursier
+  ${vars.base} {
+    name: "scalafix"
+    uri:  ${vars.uris.scalafix-uri}
+    extra.exclude: [
+      "scalafix-sbt"  // we never build sbt plugins
+      "scalafix-nsc"  // depends on circe which we don't have yet
```

Comment:
Note, most scalafix tests are currently in scalafix-nsc, core has a limited amount of tests. These tests depend on various projects (circe, shapeless,...) to reproduce bugs that surfaced in larger integration tests.